flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/connectors/hive/read/HiveMapredSplitReader.java

```diff
@@ -143,21 +139,6 @@ public HiveMapredSplitReader(
 
         // construct reuse row
         this.row = new GenericRowData(selectedFields.length);
```

Comment:
`selectedFields` will contains partition column, we should exclude partition column as we handle in extern wrapper.